bin/common.sh

```diff
@@ -106,11 +106,11 @@ fi
 function _rind() {
   local cmd="$1"
   shift
-  if [ $has_docker = 'true' ] && [ -z "$IN_DOCKER" ]; then
+  if [ $has_docker = 'true' ] && [ "${IN_DOCKER,,}" == "true" ]; then
```

Comment:
is double `,,` a syntax or typo ?
Perhaps:
`${IN_DOCKER, "false"}`

Reply:
the ${VAR,,} is used so that it will export the variable to lower case. So if someone would set IN_DOCKER=TRUE or IN_DOCKER=FALSE it would still work as we do a string comparison.